code/fallback/Hacl.IntTypes.Intrinsics.fst

```diff
@@ -9,8 +9,27 @@ open Lib.Buffer
 
 open FStar.Mul
 
-#set-options "--fuel 0 --ifuel 0 --z3rlimit 100"
+#set-options "--fuel 0 --ifuel 1 --z3rlimit 200"
```

Comment:
I usually recommend limiting ifuel to a minimum, so maybe here you could use #push-options and #pop-options for the definitions that need ifuel 1? alternatively, keep ifuel 1 and use `let _ = allow_inversion inttype` in the definitions that match on `t`